lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -60,7 +60,9 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, M
           // The error status equals -1 if the request couldn't
           // be made successfully, for example, because of a
           // timeout on an unstable network connection.
-          if (error.status === -1) {
+          const ERROR_CODE_UNSUCCESSFULL_REQUEST = -1;
```

Comment:
Yeah, I guess you could leave it as it is now, but maybe add a comment explaining that the `-1` actually relates to the `$http` object, because currently that's kinda unclear. I guess  `$http` doesn't have any useful constants you could import here?
`ERROR_CODE_UNSUCCESSFUL_REQUEST` is kinda generic-sounding ;)  What about `ERROR_HTTP_REQUEST_UNSUCCESSFUL`?

(P.S. Sorry for coming back to this so late)